docker-compose.yml

```diff
@@ -66,15 +66,14 @@ services:
       - "traefik.http.routers.pfs.tls=true"
       - "traefik.http.routers.pfs.tls.certresolver=le"
       - "traefik.http.routers.pfs.middlewares=pfs-restrict-metrics@docker"
-      - "traefik.http.middlewares.pfs-restrict-metrics.stripprefix.prefixes=/api/v1/metrics"
+      - "traefik.http.middlewares.pfs-restrict-metrics.stripprefix.prefixes=/metrics"
       - "traefik.http.routers.pfs.service=pfs"
       - "traefik.http.services.pfs.loadbalancer.healthcheck.path=/api/v1/info"
       - "traefik.http.routers.metrics-pfs.rule=Host(`metrics.pfs.${SERVER_NAME}`)"
       - "traefik.http.routers.metrics-pfs.tls=true"
       - "traefik.http.routers.metrics-pfs.tls.certresolver=le"
-      - "traefik.http.routers.metrics-pfs.middlewares=metrics-pfs-access-control@docker,metrics-pfs-metrics-path@docker"
+      - "traefik.http.routers.metrics-pfs.middlewares=metrics-pfs-access-control@docker"
```

Comment:
I don't know what this change does. I assume you know what you do.

Reply:
Thanks for the confidence ;)

Before (in `v0.12.0`) the metrics path in the services was `/api/v1/metrics`. But the default for prometheus is to expect the metrics on just `/metrics`. This middleware added that prefix.

But since it's now being exposed on the correct path the middleware is no longer necessary.